### PR TITLE
fix(suite): remove BetaBadge && align GuideButton

### LINKED ISSUE #3822: Remove BETA tag
Please: 

- [x] Remove `BETA` tag and all it's contents 
- [x] Move the floating action button with the 3D image to the bottom right corner (cca 16-24 px from the sides) 

Screen Shot 2021-06-01 at 19 29 08

### PR DESCRIPTION
**Pull request by dimasivan**
fix #3822

### CODE REVIEW

File: packages/suite/src/components/suite/SuiteLayout/index.tsx [hunk 1]
```diff
@@ -198,7 +198,6 @@ const SuiteLayout = (props: SuiteLayoutProps) => {
                     </BodyNarrow>
                 )}
             </LayoutContext.Provider>
-            <BetaBadge />
```
Comment (tomexx) [inline packages/suite/src/components/suite/SuiteLayout/index.tsx:201]: I'd recommend to remove also the component's source as well so it's not abandoned in the codebase, it's probably not needed anymore.

**Review by tomexx — CHANGES_REQUESTED** [review]

File: packages/suite/src/components/suite/SuiteLayout/index.tsx (hunk 1 above)
Comment (dimasivan) [reply packages/suite/src/components/suite/SuiteLayout/index.tsx:201]: removed with source, thx

**Review by tomexx — APPROVED** [review]

**Comment by tomasklim** [comment]
The beta badge removal should probably not be merged until the public launch release, right? @matejzak 

I added "Blocked" tag, just in case.

**Comment by glacierbits** [comment]
I suggest we merge it before the next freeze (take takes place tomorrow if I am not mistaken). It is OK if the tag is not in the app for a duration of 1 sprint. It's kind of messy and confusing next to the image button (see screenshot in the issue).

**Review by tomasklim — APPROVED** [review]

**Pull request merged**